baseball_tracker/src/App.test.js

```diff
@@ -1,9 +1,60 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-
-it('renders without crashing', () => {
-    const div = document.createElement('div');
-    ReactDOM.render(<App />, div);
-    ReactDOM.unmountComponentAtNode(div);
-});
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import "@testing-library/react/cleanup-after-each"
+import App from './App'
+
+describe('<App />', () => {
+    it('renders without crashing', () => {
+        render(<App />)
+    })
+    it('One Big Test', () => {
+        const { getByText } = render(<App />)
+        const strike = getByText(/^strike$/i)
+        const ball = getByText(/^ball$/i)
+        const foul = getByText(/^foul$/i)
+        const hit = getByText(/^hit$/i)
+
+        // Test that strike works
```

Comment:
you want to break these down into smaller individual tests, so grouped by the describe and then sectioned off into smaller test with the it(s)